typescript/sdk/src/hook/EvmHookModule.ts

```diff
@@ -927,9 +928,10 @@ export class EvmHookModule extends HyperlaneModule<
       // deploy fallback hook
       const fallbackHook = await this.deploy({ config: config.fallback });
       // deploy routing hook with fallback
-      routingHook = await this.deployer.deployContract(
+      routingHook = await this.deployer.deployContractWithName(
```

Comment:
what was this for?

Reply:
by default deployContract just uses the enum value, but here the enum value doesn't match the actual contract name (hence the type->name mapping)

its also non trivial to just change the enum because it affects other places, i hit this once before with the ism factory